lib/user-config.js

```diff
@@ -1,20 +1,16 @@
-var _ = require('lodash');
-var math = require('./math');
+const _ = require('lodash');
+const math = require('./math');
 
-var getTraffic = function(config, bot) {
-  return typeof config.botTraffic !== 'undefined' && bot ? config.botTraffic : config.traffic;
-};
-
-var pickWinner = function(features, bot) {
-  var floor = 0;
-  var rand = math.random();
-  var winner = null;
+const pickWinner = function(features) {
+  let floor = 0;
+  let rand = math.random();
+  let winner = null;
 
-  for (var name in features) {
-    var feature = features[name];
-    var traffic = getTraffic(feature, bot);
+  for (let name in features) {
```

Comment:
what is features structure like?
could you simplify this with:
```
for (traffic of features) {
....
}
```

Reply:
Features is an object.

Reply:
For/of is for iterables . . . I think that doesn't work with objects right?

Reply:
I _don't_ need the `feature` intermediate variable though.